### ISSUE TITLE
A question about symlinking songs.

### ISSUE THREAD

**Issue by MasterTextman** [stepmania/stepmania#1564]
This is not so much an issue, as it is a question [since the forums are locked].

**Can I symlink the song folder on the Linux partition with the song folder on the Windows partition?**

Let me explain - I have a HDD on my laptop with two partitions. The Linux one is the smaller one of course, and the Windows one, has a lot of data stored on it [since I used it for 1.5 years before I switched to Linux.]

However, in that time, I managed to amass ~8GB of songs and charts on Windows. I have over 1000 songs downloaded there. Now, I could very well move those songs over to the song folder in the Linux installation, but there's a big problem.

My Linux partition is very small. This is because the HDD is small in and of itself, so this partition has around 70GB at best. 8GB would fill quite a big chunk of that, so I'm curious.

Can I use the songs and charts downloaded on the Windows installation of Stepmania, with my Linux installation of stepmania - without moving said charts and songs over to the linux partition?

**Comment by concubidated** [comment]
Yes. You can totally do that.

**Comment by kyzentun** [comment]
You could also try using the AdditionalSongFolders preference.  Just put the path in Preferences.ini.

**Comment by MasterTextman** [comment]
I'll try the preferences.ini method, but just in case, how would I go about doing it the symlink way?
I read the man pages of ln and I didn't really find a way to do something, unless I didn't understand things right.

**Comment by MasterTextman** [comment]
I don't seem to have a Preferences.ini file actually.

**Comment by kyzentun** [comment]
On Linux, it's in ~/.stepmania-5.0/Save/Preferences.ini

ln follows the "<command> <source> <destination>" convention.  The source is the thing you want to link from, the destination is the path the link should be at.

This example creates a symlink from /mnt/windows/songs to /home/MasterTextman/songs.  You'll want to use the actual paths to the songs folders, don't just paste this example and expect it to work.
```
ln -s /mnt/windows/songs /home/MasterTextman/songs
```

**Comment by MasterTextman** [comment]
The thing is, I don't have a Save folder in here. I downloaded it off the main site, so that might explain it though.

Also, that actually helped out. I had to rename the Songs folder in the linux installation to something else, and go one directory above [stepmania-5.0/ instead of stepmania-5.0/Songs], but it worked. Before it kept showing me an error, or the symlink itself didn't look like a follder, but more like a file.

Thanks!
